### PR TITLE
Wizards spawn with a large satchel

### LINKED ISSUE #15821: Wizards don't spawn with a backpack
Happened a few times with me already, with the option "grey backpack" I simply didn't receive a backpack.

### PR DESCRIPTION
**Pull request by KazeEspada**
Wizards now spawn with a unique satchel. Look's like the shitty old-fashioned leather satchel, but holds as much as a duffle.

Fixes #15821

### CODE REVIEW

**Comment by newfren** [comment]
There's already a leather satchel on the wizard ship - could this be changed to be this new duffle satchel thing?

**Comment by KazeEspada** [comment]
Nah, he gets one. And one only. Unless it's in a "display case". Also, editing maps suck.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -203,6 +203,12 @@
 	icon_state = "satchel"
 	burn_state = FIRE_PROOF
 
+/obj/item/weapon/storage/backpack/satchel/wizard
+	name = "scroll case"
+	desc = "Made from the leather of \"normies\", it is superior for carrying your wizardly needs."
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/backpack.dm]: May be better to just say `assistants` as it's less OOC, but still fits.

Comment (KazeEspada) [inline code/game/objects/items/weapons/storage/backpack.dm]: It's not made from just assistants. It's made from any sort of non-magical user. IE: Normal People. Which shortens into the meme, "normies".

**Comment by Incoming5643** [comment]
Call it a bag of holding.

You know, a real one.

**Comment by KazeEspada** [comment]
Actually, the idea was handy haversack, but I wanted the handy haversack to look nicer so I just gave them a scroll case instead. SPRITES MAN, SPRITES.

File: code/game/objects/items/weapons/storage/backpack.dm (hunk 1 above)
Comment (Dorsisdwarf) [inline code/game/objects/items/weapons/storage/backpack.dm]: Is muggles copyright or something

Comment (KazeEspada) [inline code/game/objects/items/weapons/storage/backpack.dm]: I could do muggles. Which you prefer a pop culture reference or a tg station meme?

Comment (phil235) [inline code/game/objects/items/weapons/storage/backpack.dm]: I agree with Fox, the term "normies" has no place in the game.

**Comment by Cheridan** [comment]
>wizard fix
>its not a fix at all

can people stop with the inaccurate titles

**Comment by KazeEspada** [comment]
Actually it is a fix, they spawmed with no backpack and this fixes that. I just didnt think it would get stalled over a silly desc.

File: code/game/gamemodes/wizard/wizard.dm [hunk 2]
```diff
@@ -149,8 +149,7 @@
 	wizard_mob.equip_to_slot_or_del(new /obj/item/clothing/shoes/sandal(wizard_mob), slot_shoes)
 	wizard_mob.equip_to_slot_or_del(new /obj/item/clothing/suit/wizrobe(wizard_mob), slot_wear_suit)
 	wizard_mob.equip_to_slot_or_del(new /obj/item/clothing/head/wizard(wizard_mob), slot_head)
-	if(wizard_mob.backbag == 1) wizard_mob.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack(wizard_mob), slot_back)
-	if(wizard_mob.backbag == 2) wizard_mob.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/satchel_norm(wizard_mob), slot_back)
+	wizard_mob.equip_to_slot_or_del(new /obj/item/weapon/storage/backpack/satchel/wizard(wizard_mob), slot_back)
```
Comment (KazeEspada) [inline code/game/gamemodes/wizard/wizard.dm]: This is the important shit. Not the over glorified duffle below.

**Comment by KazeEspada** [comment]
@cheridan it literally says the exact issue im fixing.

File: code/game/objects/items/weapons/storage/backpack.dm (hunk 1 above)
Comment (Dorsisdwarf) [inline code/game/objects/items/weapons/storage/backpack.dm]: Normies is a 4chan meme not a /tg/station meme

**Comment by HankMoodyExperience** [comment]
>Normies is a 4chan meme not a /tg/station meme
do people not understand their roots

**Comment by Dorsisdwarf** [comment]
>Normies is a 4chan meme not a /tg/station meme
do people not understand their roots

>Implying that any significant hint of the playerbase is from /tg/

>Implying that /tg/ didn't get bored and start an actual RP station since that's what the /tg/ board likes.

**Comment by KazeEspada** [comment]
Well, it now references something else.

**Comment by Cheridan** [comment]
sorry we can't add muggles to the game we'll get sued

**Comment by KazeEspada** [comment]
God damnit.

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 3]
```diff
@@ -203,6 +203,12 @@
 	icon_state = "satchel"
 	burn_state = FIRE_PROOF
 
+/obj/item/weapon/storage/backpack/satchel/wizard
+	name = "scroll case"
+	desc = "Made from the leather of \"muggles\", it is superior for carrying your wizardly needs."
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/storage/backpack.dm]: Change it please

Comment (PKPenguin321) [inline code/game/objects/items/weapons/storage/backpack.dm]: just make it "the common folk" or something

**Comment by KorPhaeron** [comment]
@KazeEspada

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 4]
```diff
@@ -203,6 +203,12 @@
 	icon_state = "satchel"
 	burn_state = FIRE_PROOF
 
+/obj/item/weapon/storage/backpack/satchel/wizard
+	name = "scroll case"
+	desc = "Made from the leather of \"muggles\", it is superior for carrying your wizardly needs."
+	max_combined_w_class = 30
```
Comment (KorPhaeron) [inline code/game/objects/items/weapons/storage/backpack.dm]: Wait what the shit?

Comment (KazeEspada) [inline code/game/objects/items/weapons/storage/backpack.dm]: Holds as much as a duffle. IT SAID THAT IN THE DESCRIPTION THE ENTIRE TIME.

**Comment by KorPhaeron** [comment]
>3 weeks of arguing over the item desc

I'll just do this PR myself

**Pull request closed without merging**